package.json

```diff
@@ -30,9 +30,10 @@
     "wcoverage": "cross-env COVERAGE=true BROWSER=webkit node test/test.js",
     "tsc": "tsc -p .",
     "clean": "rimraf lib",
-    "build": "node utils/runWebpack.js --mode='development' && tsc -p .",
+    "build": "node utils/runWebpack.js --mode='development' && tsc -p . && npm run generate-types",
     "watch": "node utils/runWebpack.js --mode='development' --watch --silent | tsc -w -p .",
```

Comment:
Do we need something for watch?

Reply:
I'm not sure. I don't think it works right now in build though. It depends on the protocol.ts file to be generated, but the protocol generator requires the project to be built. I'll move it to a separate command that runs on bots, test, and before publishing.